main/src/library/Fixpoint/Ram/RamTerm.flix

```diff
@@ -15,6 +15,16 @@
  */
 
 namespace Fixpoint/Ram {
+
+    ///
+    /// Represents a RAM term.
+    ///
+    /// `Attribute(x, i)` represents an index into the tuple
+    /// bound to the local variable `x`, i.e. `x[i]`.
+    ///
+    /// `Lat(x)` represents the lattice component in the tuple
+    /// bound to the local variable `x`.
+    ///
     pub enum RamTerm[v] {
         case Lit(v)
         case Attribute(RowVar, Int32)
```

Comment:
Better names?

LoadVar/LoadLatVar?

ArrayLoad???

LocalVar
LocalLatVar

Reply:
I kind of like `RowLoad` and `LoadLatVar` because our local variables are named RowVar and represent rows in a database relation. ArrayLoad is a bit specific to our current implementation with arrays as tuples.